src/api/app/models/project.rb

```diff
@@ -968,6 +976,44 @@ def branch_to_repositories_from(project, pkg_to_enable, opts = {})
     end
   end
 
+  def branch_remote_repositories(project)
+    remote_project = RemoteProject.new(name: project)
+    remote_project_meta = Nokogiri::XML(remote_project.meta.to_s)
+    local_project_meta = Nokogiri::XML(to_axml)
+
+    remote_repositories = remote_project.repositories_from_meta
+    remote_repositories -= repositories.where(name: remote_repositories).pluck(:name)
+
+    remote_repositories.each do |repository|
```

Comment:
How is this easier than [RepositoryController#create](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/controllers/webui/repositories_controller.rb#L38)?